src/dndbeyond/content-scripts/source.js

```diff
@@ -42,7 +42,7 @@ function documentLoaded(settings) {
             injectDiceToRolls(".primary-content", character, (node) => {
                 return nearestHeading(node, source_name);
             });
-            const read_aloud = $(".read-aloud-text,.adventure-read-aloud-text");
+            const read_aloud = $(".read-aloud-text,.adventure-read-aloud-text,.text--quote-box");
```

Comment:
I'm not sure if I'd put `aside.text--quote-box` to be more specific, or if I'd put it the way you did so it's more inclusive (in case they use something other than `aside` somewhere else, and "quote-box" seems to be a descriptive enough class...) 🤔

Reply:
Dude, who knows, they might change every single thing tomorrow again, the way they're going...

Reply:
lol, very true! 😁 
Anyways, I thought on it for a bit and I think it's better without the `aside` anyway, so will merge it now, thanks!